doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -404,11 +404,13 @@ Field           | Type       | Meaning
 
 The `mz_dataflow_channel_operators` view associates [dataflow] channels with the operators that are their endpoints.
 
-Field              | Type       | Meaning
--------------------|------------|--------
-`id`               | [`bigint`] | The ID of the channel. Corresponds to [`mz_dataflow_channels.id`](#mz_dataflow_channels).
-`from_operator_id` | [`bigint`] | The ID of the source of the channel. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).
-`to_operator_id`   | [`bigint`] | The ID of the target of the channel. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).
+| Field                   | Type            | Meaning                                                                                                             |
+|-------------------------|-----------------|---------------------------------------------------------------------------------------------------------------------|
+| `id`                    | [`bigint`]      | The ID of the channel. Corresponds to [`mz_dataflow_channels.id`](#mz_dataflow_channels).                           |
+| `from_operator_id`      | [`bigint`]      | The ID of the source of the channel. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).           |
+| `from_operator_address` | [`bigint list`] | The address of the source of the channel. Corresponds to [`mz_dataflow_addresses.address`](#mz_dataflow_addresses). |
+| `to_operator_id`        | [`bigint`]      | The ID of the target of the channel. Corresponds to [`mz_dataflow_operators.id`](#mz_dataflow_operators).           |
+| `to_operator_address`   | [`bigint list`] | The address of the target of the channel. Corresponds to [`mz_dataflow_addresses.address`](#mz_dataflow_addresses). |
```

Comment:
Nit: The formatting style of this table now is different from all other tables in this doc. No idea if the docs team cares, but it bugs me!

Reply:
Actually, there are some tables formatted this way already (`mz_dataflow_arrangement_sizes`), and some the other way, so this is not adding any inconsistency.

My editor uses this format automatically; unfortunately it doesn't seem to be configurable.

Reply:
I have reformatted all tables in this style, for consistency.